src/App.tsx

```diff
@@ -112,11 +112,6 @@ const App: React.FC<{}> = () => {
               walletSelect: {
                 wallets: [
                   { walletName: "metamask", preferred: true },
-                  {
-                    walletName: "walletConnect",
-                    preferred: true,
-                    rpc,
-                  },
```

Comment:
Why don't we need to remove the relevant dependencies and imports?

Reply:
Thank you for your question!
WalletConnect v1 is not a library and does not have any dependencies. It is simply part of the configuration for `Onboard`.